source/ranges.tex

```diff
@@ -9520,7 +9520,7 @@
     @\exposid{iterator}@() = default;
     constexpr @\exposid{iterator}@(@\exposid{iterator}@<!Const> i)
       requires Const && (@\libconcept{convertible_to}@<iterator_t<Views>,
-                                        iterator_t<@\exposid{maybe-const}@<Const, Views>>> && ...);
+                                        iterator_t<const Views>> && ...);
```

Comment:
Can we reformat so that "convertible_to" is on its own (single) line, like the other cases in this pull request?